src/main/java/core/basesyntax/strategy/ReturnOperationHandler.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.dao.ProductDao;
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperationHandler implements OperationHandler {
+    private final ProductDao productDao;
+
+    public ReturnOperationHandler(ProductDao productDao) {
+        this.productDao = productDao;
+    }
+
+    @Override
+    public void operationByTransaction(FruitTransaction fruitTransaction) {
```

Comment:
I haven't enough experience to help here but maybe rename this method to doOperation... , or makeOperation...... or etc :)